### ISSUE TITLE
How do I know I'm up-to-date?

### ISSUE THREAD

**Issue by dmorcellet** [LotroCompanion/lotro-companion#509]
Garlo dixit:
Could we have a check for updates button or a "you are up to date" type message on the about panel?
Currently I don't feel like i ever know if i'm up to date or not.
or Updates: Last checked 2023-06-26 19:33

(the app looks for updates on startup, but this may fail silently).

**Comment by dt192** [comment]
Yes, I think a `last successful check date` as mentioned above would be good and a pop up alert if the app fails to contact the update server at all.
